delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/client/ElasticSearchClient.scala

```diff
@@ -32,17 +36,25 @@ import scala.reflect.ClassTag
   */
 class ElasticSearchClient(client: HttpClient, endpoint: Uri)(implicit as: ActorSystem) {
   import as.dispatcher
-  private val serviceName                                        = Name.unsafe("elasticsearch")
-  private val docPath                                            = "_doc"
-  private val allIndexPath                                       = "_all"
-  private val bulkPath                                           = "_bulk"
-  private val ignoreUnavailable                                  = "ignore_unavailable"
-  private val allowNoIndices                                     = "allow_no_indices"
-  private val searchPath                                         = "_search"
-  private val newLine                                            = System.lineSeparator()
-  private val `application/x-ndjson`: MediaType.WithFixedCharset =
+  private val logger: Logger                                        = Logger[ElasticSearchClient]
+  private val serviceName                                           = Name.unsafe("elasticsearch")
+  private val scriptPath                                            = "_scripts"
+  private val docPath                                               = "_doc"
+  private val allIndexPath                                          = "_all"
+  private val bulkPath                                              = "_bulk"
+  private val tasksPath                                             = "_tasks"
+  private val waitForCompletion                                     = "wait_for_completion"
+  private val ignoreUnavailable                                     = "ignore_unavailable"
+  private val allowNoIndices                                        = "allow_no_indices"
+  private val updateByQueryPath                                     = "_update_by_query"
+  private val searchPath                                            = "_search"
+  private val newLine                                               = System.lineSeparator()
+  private val `application/x-ndjson`: MediaType.WithFixedCharset    =
     MediaType.applicationWithFixedCharset("x-ndjson", HttpCharsets.`UTF-8`, "json")
-  private val defaultQuery                                       = Map(ignoreUnavailable -> "true", allowNoIndices -> "true")
+  private val defaultQuery                                          = Map(ignoreUnavailable -> "true", allowNoIndices -> "true")
```

Comment:
On the way indices are managed by the stream, indices are always created before bulk/search/updates.
This fields are present to avoid managing some errors (for example when having an aggregatedElasticSearchView where the client has access to nothing, or some edge cases related to querying).

This is something that existed before and this PR is not doing anything to change it.